README.md

```diff
@@ -13,4 +13,15 @@
 4. `bundle exec rails s` 
 5. Obtain a copy of the 'commitments' CSV if not already present in `lib/data/seeds`
 6. Run `rake import:commitments`
-7. On the Commitments page, you should see a list of commitments.
\ No newline at end of file
+7. On the Commitments page, you should see a list of commitments.
+
+### Adding new commitments via csv
+We are dependant on several fields on Commitment to filter the results we want to return over the API.
+```scope :api_records, -> { where(state: 'live', cfn_approved: true, shareable: true) }```
```

Comment:
Is this code meant to be on a separate line? If so, I think it might the ``` to be on separate lines (or maybe an extra line break before)? Currently the code snippet is tagging onto the end of the previous line: https://github.com/unepwcmc/action-agenda/tree/feat/basic_api